go.mod

```diff
@@ -4,9 +4,14 @@ require (
 	github.com/fxamacker/cbor/v2 v2.2.0
 	github.com/go-kit/kit v0.8.0
 	github.com/go-logfmt/logfmt v0.4.0 // indirect
+	github.com/go-playground/locales v0.13.0
+	github.com/go-playground/universal-translator v0.17.0
 	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/google/uuid v1.1.0
+	github.com/leodido/go-urn v1.1.0 // indirect
 	github.com/stretchr/testify v1.3.0
+	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
+	gopkg.in/go-playground/validator.v9 v9.31.0
```

Comment:
The current version of this library is [v10.2.0](https://github.com/go-playground/validator/releases/tag/v10.2.0). Recommend seeing if that works. Looking at the `go.sum` you're still going to have a problem though with the transitive dependency to [pmezard/go-difflib](https://github.com/pmezard/go-difflib) above.

Reply:
Updated [go-playground/validator](https://github.com/go-playground/validator) to v10.2.0 and it's still using the [pmezard/go-difflib](https://github.com/pmezard/go-difflib) module as you pointed out. 
Did some investigation but found no other good alternatives for the validator feature.

Reply:
mezard/go-difflib is a small and simple module, and it is an indirect dependency.  go-playground/validator should adjust it if there is any trouble happening in the future.  We can discuss in the WG meeting to see whether we can accept it.